src/command.js

```diff
@@ -40,7 +40,14 @@ class GraphqurlCommand extends Command {
     }
 
     if (queryString === null) {
-      queryString = await getQueryFromTerminalUI(endpoint, headers);
+      queryString = await getQueryFromTerminalUI(endpoint, headers)
+      .catch(error => {
```

Comment:
@TFarla please handle this error inside `ui.js`. There is no context for introspection here in `command.js`.